### PR TITLE
add the note about 4 byte support

### PR DESCRIPTION
**Pull request by jospoortvliet**
So ppl don't miss it... Including me...

### CODE REVIEW

File: admin_manual/maintenance/backup.rst [hunk 1]
```diff
@@ -48,6 +48,10 @@ database engine. To backup MySQL/MariaDB::
 
     mysqldump --single-transaction -h [server] -u [username] -p[password] [db_name] > nextcloud-sqlbkp_`date +"%Y%m%d"`.bak
 
+If you use `4 byte support <https://docs.nextcloud.com/server/latest/go.php?to=mysql_4byte_support`_ in MySQL/MariaDB (needed for emoji), you will need to add ``--default-character-set=utf8mb4`` like this:
```
Comment (MorrisJobke) [inline admin_manual/maintenance/backup.rst:51]:
```suggestion
If you use `4 byte support <https://docs.nextcloud.com/server/latest/go.php?to=mysql_4byte_support`_ in MySQL/MariaDB (needed for emoji), you will need to add ``--default-character-set=utf8mb4`` like this::
```

**Review by MorrisJobke — APPROVED** [review]

File: admin_manual/maintenance/backup.rst [hunk 2]
```diff
@@ -48,6 +48,10 @@ database engine. To backup MySQL/MariaDB::
 
     mysqldump --single-transaction -h [server] -u [username] -p[password] [db_name] > nextcloud-sqlbkp_`date +"%Y%m%d"`.bak
 
+If you use `4 byte support <https://docs.nextcloud.com/server/latest/go.php?to=mysql_4byte_support`_ in MySQL/MariaDB (needed for emoji), you will need to add ``--default-character-set=utf8mb4`` like this::
```
Comment (nickvergessen) [inline admin_manual/maintenance/backup.rst:51]: nöd

**Comment by nickvergessen** [comment]
STOP

**Comment by nickvergessen** [comment]
-.-

File: admin_manual/maintenance/backup.rst (hunk 2 above)
Comment (nickvergessen) [reply admin_manual/maintenance/backup.rst:51]: Unknown target name: "4 byte support <https://docs.nextcloud.com/server/latest/go.php?to=mysql_4byte_support".

Comment (jospoortvliet) [reply admin_manual/maintenance/backup.rst:51]: what does nod mean?

**Pull request merged**